tests/rule_based_profiler/parameter_builder/test_numeric_metric_range_multi_batch_parameter_builder.py

```diff
@@ -237,3 +244,218 @@ def test_oneshot_numeric_metric_range_multi_batch_parameter_builder_bobby(
+            variables=variables,
+            parameters=parameters,
+            batch_request=batch_request,
+        )
+
+
+def test_bootstrap_numeric_metric_range_multi_batch_parameter_builder_bobby_false_positive_rate_zero(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    data_context: DataContext = (
+        bobby_columnar_table_multi_batch_deterministic_data_context
+    )
+
+    # BatchRequest yielding three batches
+    batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder
+    numeric_metric_range_parameter_builder = (
+        NumericMetricRangeMultiBatchParameterBuilder(
+            name="row_count_range",
+            metric_name="table.row_count",
+            estimator="bootstrap",
+            false_positive_rate=0.0,
+            round_decimals=0,
+            data_context=data_context,
+        )
+    )
+
+    variables: Optional[ParameterContainer] = None
+
+    domain: Domain = Domain(
+        domain_type=MetricDomainTypes.TABLE,
+    )
+    parameter_container: ParameterContainer = ParameterContainer(parameter_nodes=None)
+    parameters: Dict[str, ParameterContainer] = {
+        domain.id: parameter_container,
+    }
+
+    assert parameter_container.parameter_nodes is None
+
+    warning_message: str = re.escape(
+        f"""You have chosen a false_positive_rate of 0.0, which is too close to 0.
+A false_positive_rate of {NP_EPSILON} has been selected instead."""
+    )
+
+    with pytest.warns(UserWarning, match=warning_message):
+        numeric_metric_range_parameter_builder.build_parameters(
+            domain=domain,
+            variables=variables,
+            parameters=parameters,
+            batch_request=batch_request,
+        )
+
+
+def test_bootstrap_numeric_metric_range_multi_batch_parameter_builder_bobby_false_positive_rate_very_small(
+    bobby_columnar_table_multi_batch_deterministic_data_context,
+):
+    data_context: DataContext = (
+        bobby_columnar_table_multi_batch_deterministic_data_context
+    )
+
+    # BatchRequest yielding three batches
+    batch_request: dict = {
+        "datasource_name": "taxi_pandas",
+        "data_connector_name": "monthly",
+        "data_asset_name": "my_reports",
+    }
+
+    # a commonly used defect rate in quality control that equates to 3.4 defects per million opportunities
+    six_sigma_false_positive_rate: float = 3.4 / 1000000.0
+    assert six_sigma_false_positive_rate > NP_EPSILON
+
+    # what if user tries a false positive rate smaller than NP_EPSILON (by an order of magnitude in this case)?
+    smaller_than_np_epsilon_false_positive_rate: float = NP_EPSILON / 10
+
+    numeric_metric_range_parameter_builder: NumericMetricRangeMultiBatchParameterBuilder
+    numeric_metric_range_parameter_builder = (
```

Comment:
@NathanFarmer there is only one here...  So the type hint and the assignment can be one line.  Thanks.

Reply:
@NathanFarmer I would advocate that we accept the "line too long" complaint in favor of not splitting the type hint / variable declaration lines in the case that the given variable occurs only once (like it was before).  Thanks!

Reply:
@alexsherstinsky I have reverted the change